change-beta/@azure-communication-react-1e9ee08b-b0d6-4834-adc3-3e041756647f.json

```diff
@@ -0,0 +1,9 @@
+{
+  "type": "patch",
```

Comment:
nit: technically a none change as this only impacts dev tooling and doesn't impact the published package in any way